packages/vite/src/node/plugins/resolve.ts

```diff
@@ -505,13 +504,35 @@ export function tryNodeResolve(
   const nestedRoot = id.substring(0, lastArrowIndex).trim()
   const nestedPath = id.substring(lastArrowIndex + 1).trim()
 
-  // check for deep import, e.g. "my-lib/foo"
-  const deepMatch = nestedPath.match(deepImportRE)
```

Comment:
I think this no longer needs to be exported from `utils`. Not sure if it's worth changing that there or not